concourse/client.py

```diff
@@ -77,6 +78,8 @@ class SetPipelineResult(Enum):
 
 # GLOBAL DEFINES
 CONCOURSE_API_SUFFIX = 'api/v1'
+AUTH_TOKEN_REQUEST_USER = 'fly'
+AUTH_TOKEN_REQUEST_PWD = 'Zmx5'
```

Comment:
this feels so very, very wrong. could you at least add a comment linking to the concourse sources from where this was copied?

also, this might be better kept as attribute of model/concourse.py (ConcourseConfig); this way, it is at least somewhat prepared to be configurable (just in case it changes again in the future; could happen with each new cc version, could it not?)

Reply:
Done